draft-ietf-webtrans-http3.md

```diff
@@ -265,6 +265,54 @@ within server-initiated bidirectional streams is WEBTRANSPORT_STREAM.
 TODO: move the paragraph above into a separate draft; define what happens with
 already existing HTTP/3 frames on server-initiated bidirectional streams.
 
+## Resetting Data Streams
+
+A WebTransport endpoint may send a RESET_STREAM or a STOP_SENDING frame for a
+WebTransport data stream.  Those signals are propagated by the WebTransport
+implementation to the application.
+
+A WebTransport application SHALL provide an error code for those operations.
+Since WebTransport shares the error code space with HTTP/3, WebTransport
+application errors for streams are limited to an unsigned 8-bit integer,
+assuming values between 0x00 and 0xff.  WebTransport implementations SHALL
+remap those error codes into an error range where 0x00 corresponds to
+0x52e4a40fa8db, and 0xff corresponds to 0x52e4a40fa9e2.  Note that there are
+code points inside that range of form "0x1f * N + 0x21" that are reserved by
+{{Section 8.1 of HTTP3}}; those have to be accounted for when mapping the error
```

Comment:
Great to have pseudocode, is there a chance we could explain _how_ someone is expected to account for those (i.e. do you just consider those invalid/taken already or shift by that much each time you pass an already reserved one) in text so that when people get to the pseudocode they know what behavior they're supposed to be seeing?

Reply:
You account by shifting.  Clarified that.